package.json

```diff
@@ -120,7 +120,7 @@
     "i18n:compile": "lingui compile",
     "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'",
     "postinstall": "yarn compile-contract-types",
-    "start": "yarn compile-contract-types && react-scripts start",
+    "start": "yarn compile-contract-types && yarn i18n:extract && react-scripts start",
```

Comment:
summoning @moodysalem 

I also prefer generating on start. It does slow down starting a server so could also be a postinstall or something similar.

Reply:
if you just do a `touch` does it also work?